src/resources/views/crud/form_content.blade.php

```diff
@@ -5,7 +5,7 @@
     @include('crud::inc.show_tabbed_fields')
     <input type="hidden" name="_current_tab" value="{{ Str::slug($crud->getTabs()[0]) }}" />
 @else
-  <div class="card">
+  <div class="card border-0">
     <div class="card-body row">
       @include('crud::inc.show_fields', ['fields' => $crud->fields()])
     </div>
```

Comment:
Note that TinyMCE is broken. But that should NOT be fixed on this branch. Don't worry about it @maurohmartinez , Antonio will fix that on the `v6` branch. It's probably because of the upgrade to the latest TinyMCE.

![CleanShot 2023-02-26 at 11 05 08](https://user-images.githubusercontent.com/1032474/221401567-e6e4138f-0bb1-454e-bb74-8c36014b81d0.png)

Reply:
Note - since this is happening on the `v6` branch too, this isn't something you should fix Mauro. I'll ask Antonio to fix it on v6 - https://github.com/Laravel-Backpack/PRO/pull/123/files#r1118047306

Reply:
Fixed on `v6`. I just pulled the fix to the `coreuiv4` branch too.